pkg/config/filter_test.go

```
@@ -0,0 +1,715 @@
+		},
+		{
+			name: "sample exemption: non-redhat email passes via notmatches",
+			filter: &InvestigationFilter{
+				Filter: &FilterNode{
+					Or: []FilterNode{
+						{Field: "OwnerEmail", Operator: OperatorNotMatches, Values: []string{".*@redhat\\.com$"}},
+						{Operator: OperatorSample, Values: []string{"0"}},
+					},
+				},
+			},
+			ctx: &types.FilterContext{OwnerEmail: "user@example.com"},
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofmt)